Assets/MRTK/Core/Utilities/Editor/EditorProjectUtilities.cs

```diff
@@ -19,6 +19,8 @@ static EditorProjectUtilities()
         {
             CheckMinimumEditorVersion();
             ApplyARFoundationUWPCompileFix();
+            MixedRealityToolkitPreserveSettings.EnsureLinkXml();
```

Comment:
According to Unity's own docs: https://docs.unity3d.com/Manual/ManagedCodeStripping.html

It looks like it only supports one link.xml for the entire project (or really, it just says it's per-project) - if the consumer has their own link.xml does this still end up working out, or does one of them end up taking over the other?

Reply:
We're good on this.

Per: https://docs.unity3d.com/Manual/ManagedCodeStripping.html#LinkXML
"You can use any number of link.xml files in a Project, so plugins can provide their own preservation declarations."

Reply:
That doc seems slightly contradictory (or at least, could use better terminology). A few sentences after they call it "per-Project", they say:

>You can use any number of link.xml files in a Project, so plugins can provide their own preservation declarations.